editor/js/Menubar.File.js

```diff
@@ -38,19 +38,78 @@ function MenubarFile( editor ) {
 	options.setClass( 'options' );
 	container.add( options );
 
-	// New
+	// New Project
 
 	var option = new UIRow();
 	option.setClass( 'option' );
 	option.setTextContent( strings.getKey( 'menubar/file/new' ) );
 	option.onClick( function () {
-
+		
 		if ( confirm( 'Any unsaved data will be lost. Are you sure?' ) ) {
+			
+			editor.clear();
+			config.setKey( 'project/title', 'untitled' );
+			
+		}
+	} );
+	options.add( option );
+
+	// Load Project
+
+	var openform = document.createElement( 'form' );
+	openform.style.display = 'none';
+	document.body.appendChild( openform );
+
+	var openfileInput = document.createElement( 'input' );
+	openfileInput.multiple = false;
+	openfileInput.type = 'file';
+	openfileInput.accept = '.json';
+	openfileInput.addEventListener( 'change', function () {
+		
+		editor.loader.loadFiles( openfileInput.files );
+		openform.reset();
+		
+	} );
+	openform.appendChild(openfileInput);
 
+	var option = new UIRow();
+	option.setClass( 'option' );
+	option.setTextContent( strings.getKey( 'menubar/file/load' ) );
+	option.onClick( function() {
+		
+		if ( confirm( 'Any unsaved data will be lost. Are you sure?' ) ) {
+			
 			editor.clear();
+			openfileInput.click();
```

Comment:
Um, it feels a bit strange that the editor is already cleared at this point. I guess it's better when the clear should only happen if the user actually selects a JSON from the input dialog.

Reply:
Good observation